interface-elements-for-web/articles/diagram/work-with-chart-elements.md

```diff
@@ -11,20 +11,30 @@ You can use the mouse or keyboard commands to move an element or a group of [sel
 - **Mouse**: drag and drop an element. 
 - **Keyboard**: select an element and press arrow keys to move the element.
 
-If the [Snap to Grid](page-and-view-settings.md#grid-settings) option is enabled, an element is moved snapping to the grid. Press and hold Ctrl to move the element smoothly.
+Press and hold Shift to move the element horizontally or vertically.
+
+![Mobile Element](../../images/diagram-move-element-shift.gif)
+
+If the [Snap to Grid](page-and-view-settings.md#grid-settings) option is enabled, an element is moved snapping to the grid. Press and hold Ctrl to move the element smoothly. 
 
 ![Mobile Element](../../images/diagram-move-element.gif)
 
+You can combine the Ctrl and Shift buttons.
+
 ## Resize a Shape
 
 1. [Select a shape](select-elements.md) to display resizing squares along the selection.
 
-1. Drag a square to resize the shape. 
+1. Drag a square to resize the shape.
+
+Press and hold Shift to maintain the shape's width-to-height ratio.
 
 If the [Snap to Grid](page-and-view-settings.md#grid-settings) option is enabled, a shape is resized snapping to the grid. Press and hold Ctrl to resize the element smoothly.
 
 ![Resize a Shape](../../images/diagram-resize-shape.gif)
 
+You can combine the Ctrl and Shift buttons.
+
 ## Transform a Shape
 
 Certain shapes display a transform handle when selected. Use this handle to transform the shape.

```

Comment:
transform handle? Did you mean a Transformation handle?

Reply:
```suggestion
Certain shapes display a transformation handle when selected. Use this handle to transform the shape.

```

Reply:
ok